File: support_tools/swigwin-3.0.12/Examples/test-suite/go/nested_workaround_runme.go
package main

import . "./nested_workaround"

func main() {
	inner := NewInner(5)
	outer := NewOuter()
	newInner := outer.DoubleInnerValue(inner)
	if newInner.GetValue() != 10 {
		panic(0)
	}

	outer = NewOuter()
	inner = outer.CreateInner(3)
	newInner = outer.DoubleInnerValue(inner)
	if outer.GetInnerValue(newInner) != 6 {
		panic(0)
	}
}

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/nested_workaround_runme.py
from nested_workaround import *

inner = Inner(5)
outer = Outer()
newInner = outer.doubleInnerValue(inner)
if newInner.getValue() != 10:
    raise RuntimeError

outer = Outer()
inner = outer.createInner(3)
newInner = outer.doubleInnerValue(inner)
if outer.getInnerValue(newInner) != 6:
    raise RuntimeError
